### DISCUSSION TITLE
tutorial how to install allsky-modules /allsky_dewheater

### DISCUSSION THREAD

**Discussion by sergio1932** [AllskyTeam/allsky-modules#175] [General]
Good night.

I try to install the script for 
allsky_dewheater and I have no idea how to install it. I don't have much of an idea and I'm a little frustrated. I don't know how to manage that .py, is there a tutorial on how to install it and have it available on the web? thanks in advance

**Comment by Alex-developer** [comment]
Yes, have a read of this page https://github.com/AllskyTeam/allsky-modules it explains howto install the modules. Its an automated process once you have cloned the repository.
